docs/configuration/config-file-reference.md

```diff
@@ -153,213 +153,8 @@ tenant_federation:
 # The ruler_config configures the Cortex ruler.
 [ruler: <ruler_config>]
 
-ruler_storage:
```

Comment:
Moved below to its dedication section to keep this huge reference file cleaner.

Reply:
👍